js/StatusBar.ios.js

```diff
@@ -11,7 +11,7 @@
 
 'use strict';
 
-import { NativeEventEmitter, NativeModules as Modules } from 'react-native'
+import { NativeEventEmitter, NativeModules as Modules } from 'react-native';
```

Comment:
Remove the whitespace:

```
import {NativeEventEmitter, NativeModules as Modules} from 'react-native';
```

Reply:
why?

Reply:
I just thought abought making every imports using whitespaces...

Reply:
Could you use this eslint config and reformat the project? https://www.npmjs.com/package/@react-native-community/eslint-config

We are trying to make sure all community projects use the same config as every other RN project.

Reply:
I am using it

Reply:
Oh, could you also add this: https://github.com/facebook/react-native/blob/master/package.json#L13-L20 so it will format same as RN?